.github/actions/shared-steps/action.yml

```diff
@@ -101,6 +122,22 @@ runs:
 
         echo "image-id=sha256:$(sudo podman image inspect ${{ env.IMAGE_NAME }} -f "{{.Id}}")" >> $GITHUB_OUTPUT
 
+    - name: Build Image Tars
```

Comment:
What is the purpose of this step?

Reply:
I like that, but it should probably be in a different PR

Reply:
Removed. Will make a followon PR